lib/Doctrine/ORM/Proxy/ProxyFactory.php

```diff
@@ -181,6 +292,8 @@ protected function createProxyDefinition($className)
     /**
      * Creates a closure capable of initializing a proxy
      *
+     * @deprecated ProxyFactory::createInitializer() is deprecated and will be removed in version 3.0 of doctrine/orm.
```

Comment:
Do we need to deprecate a private method?

Reply:
I added those to help when cleaning for 3.0. Let me know.